cvat-ui/src/actions/boundaries-actions.ts

```diff
@@ -43,6 +43,8 @@ export function resetAfterErrorAsync(): ThunkAction {
             const state = getState();
             const job = state.annotation.job.instance;
 
+            state.annotation.canvas.instance.destroy();
```

Comment:
The same comment, not very clear, why we call destroy() here.

Reply:
Moved it back to reducer